### PR TITLE
Feature 154. See events on restaurant page. narek

### PR DESCRIPTION
**Pull request by narekacy**

### CODE REVIEW

**Comment by vitaliimak** [comment]
@narekacy could you merge current dev branch into your feature branch?

**Review by r-yevchuk — DISMISSED** [review]

File: client/src/components/restaurant/restaurant-item.js [hunk 1]
```diff
@@ -85,7 +86,7 @@ class Restaurant extends Component {
             <Menu id={id} isAuthenticated={isAuthenticated} />
           </Tab>
           <Tab eventKey="events" title="Events">
-            <h3>Events</h3>
+            <RestaurantEvents id={id} />
```
Comment (vitaliimak) [inline client/src/components/restaurant/restaurant-item.js:101]: This link doesn't need on this page, could you remove it?
image

Comment (VozNick) [reply client/src/components/restaurant/restaurant-item.js:101]: > This link doesn't need on this page, could you remove it?
> image

@narekacy no answer to this comment?) link is removed or will stay the same?

Comment (narekacy) [reply client/src/components/restaurant/restaurant-item.js:101]: Sorry, just wanted to remove it and answer that it is done and already pushed. 
Currently I'm working on task #137 as it is important one for this sprint.

I see all comments, sorry for late answers.

**Review by VozNick — DISMISSED** [review]

File: client/src/components/restaurant/restaurant-item.js (hunk 1 above)
Comment (narekacy) [reply client/src/components/restaurant/restaurant-item.js:101]: Done

Comment (vitaliimak) [reply client/src/components/restaurant/restaurant-item.js:101]: By default, links in Internet have blue color. This style can be misleading users
image

Comment (vitaliimak) [reply client/src/components/restaurant/restaurant-item.js:101]: Don't remove link from **all events** page. Only from **restaurant events** page (it leads to the same page)
image

**Review by vitaliimak — CHANGES_REQUESTED** [review]

File: client/src/components/restaurant/restaurant-item.js (hunk 1 above)
Comment (narekacy) [reply client/src/components/restaurant/restaurant-item.js:101]: I thiink events link shouldn't move user to restaurant page but for event page. According to absense of event page (full card page) there is no neccessary to move user to restaurant page.

Comment (narekacy) [reply client/src/components/restaurant/restaurant-item.js:101]: > By default, links in Internet have blue color. This style can be misleading users

Done

Comment (vitaliimak) [reply client/src/components/restaurant/restaurant-item.js:101]: > I thiink events link shouldn't move user to restaurant page but for event page. According to absense of event page (full card page) there is no neccessary to move user to restaurant page.

Now we use this functionality, you shouldn't break current functionality
If you want to add event page, you can do special task for it

Comment (narekacy) [reply client/src/components/restaurant/restaurant-item.js:101]: I'm not about adding event page right now)

I say moving user to restaurant page isn't UX that user expects. He awaits to see information about event. So we missed this when this functionality was added and I think nothing is breaked while improved.

**Review by r-yevchuk — DISMISSED** [review]

**Review by VozNick — DISMISSED** [review]

**Review by vitaliimak — APPROVED** [review]

**Review by VozNick — APPROVED** [review]

**Review by r-yevchuk — APPROVED** [review]

**Pull request merged**